packages/core/src/Stores/pushwoosh-store.js

```diff
@@ -5,8 +5,8 @@ import { getLanguage } from '@deriv/translations';
 import BaseStore from './base-store';
 
 export default class PushwooshStore extends BaseStore {
-    // only available on production (bot and deriv)
-    is_applicable = /^(16929|19111|24091)$/.test(getAppId());
+    // only available on staging & production (bot and deriv)
+    is_applicable = /^(16929|19111|24091|16303)$/.test(getAppId());
     has_initialized = false;
     push_woosh = new Pushwoosh();
```

Comment:
Won't `push_woosh` be undefined in local and test environments if it's only available on staging & production?

Reply:
this is class from web-push-notification sdk, we dont get this from window, so the class interface will still be available